### ISSUE TITLE
How to prevent completion pop-up on space character with recommended omnisharp-vim setting?

### ISSUE THREAD

**Issue by Ghopper21** [Shougo/neocomplete.vim#443]
Apologies if this isn't the right place to ask -- I couldn't find any email forum and I've read through the help doc and tried various things myself, without success.

My problem is that with a cs file using omnisharp-vim completion and the following setting (recommended by omnisharp-vim [here](https://github.com/OmniSharp/omnisharp-vim/wiki/Example-NeoComplete-Settings)):

    let g:neocomplete#sources#omni#input_patterns.cs = '.*[^=\);]'

I'm getting a completion pop-up (with a very very long list of completions, basically every possible string) after a single space character. I've tried changing the pattern to '\S*[^=\);\s]' with the same behavior. 

I'm sure this is just a matter of getting the right setting -- any help would be much appreciated!

P.S. THANKS for a great plugin, and many other plugins also!

**Comment by Shougo** [comment]
It is just keyword pattern problem.
What's your desired pattern?

```vim
let g:neocomplete#sources#omni#input_patterns.cs = '.*[^=\);]'
```

Above pattern matches every characters.  I don't recommend to use it.

**Comment by Ghopper21** [comment]
Thanks. I'm not sure what the ideal pattern for C# is, but yes this clearly
isn't right.

I'd like the neocomplete pop-up to appear with OmniSharp completion results
whenever I've typed one or more characters that could part of an
identifier, i.e. alphanumeric or underscore.

I've tried `'[a-zA-Z0-9_]+'` but then OmniSharp completions do NOT show up
for anything.

**Comment by Ghopper21** [comment]
Thanks. I'm not sure what the ideal pattern for C# is, but yes this clearly isn't right.

I'd like the neocomplete pop-up to appear with OmniSharp completion results whenever I've typed one or more characters that could part of an identifier, i.e. alphanumeric or underscore.

I've tried:

    let g:neocomplete#sources#omni#input_patterns.cs = '[a-zA-Z0-9_]+'

but then OmniSharp completions do NOT show up for anything.
